### PR TITLE
Add animal content

### PR DESCRIPTION
**Pull request by lindaicing**
Added Animal Content that displays conditionally
Screen Shot 2020-08-23 at 12 38 43 AM
Screen Shot 2020-08-23 at 12 39 32 AM

### CODE REVIEW

**Comment by CubeOfCheese** [comment]
When there are no certifications that means the score was manually produced so it could say that like it used to

**Comment by CubeOfCheese** [comment]
Maybe the list of sources should be indented, bulleted, or numbered. When only one shows up (which will be the most common result) it took me a second to realize that was a listed source

File: Website-and-Backend/src/main/resources/public/info.js [hunk 1]
```diff
@@ -27,24 +27,13 @@ function renderPage(companyData) {
...
-  if (companyData.leapingBunnyCertified) {
-    document.getElementById("leapingbunny").style = "display:block;"
-  }
+
+  if (companyData.bluesignPartner) { document.getElementById("bluesign").style = "display:block;" }
+  if (companyData.bcorpCertified) { document.getElementById("bcorp").style = "display:block;" }
+  if (companyData.blackOwnedBusiness) { document.getElementById("blackowned").style = "display:block;" }
+  if (companyData.supportsBLM) { document.getElementById("blmsupport").style = "display:block;" }
+  if (companyData.veganDotOrgCertified) { document.getElementById("vegan").style = "display:block;" }
+  if (companyData.leapingBunnyCertified) { document.getElementById("leapingbunny").style = "display:block;" }
```
Comment (CubeOfCheese) [inline Website-and-Backend/src/main/resources/public/info.js:36]: Wow I like this style a lot better than what I was doing before with one line if clauses.

**Comment by lindaicing** [comment]
> When there are no certifications that means the score was manually produced so it could say that it was manually produced

But sometimes scores show up like this 
Screen Shot 2020-08-23 at 8 34 04 PM

**Comment by lindaicing** [comment]
> Maybe the list of sources should be indented, bulleted, or numbered. When only one shows up (which will be the most common result) it took me a second to realize that was a listed source

Screen Shot 2020-08-23 at 10 21 21 PM

**Comment by CubeOfCheese** [comment]
> > When there are no certifications that means the score was manually produced so it could say that it was manually produced
> 
> But sometimes scores show up like this
> Screen Shot 2020-08-23 at 8 34 04 PM

I don't understand what you are trying to say

**Pull request merged**